src/data/thunkActions/grades.js

```diff
@@ -0,0 +1,226 @@
+/* eslint-disable import/no-self-import */
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+
+import { StrictDict } from 'utils';
+import grades from '../actions/grades';
+import { sortAlphaAsc } from '../actions/utils';
+
+import selectors from '../selectors';
+
+import GRADE_OVERRIDE_HISTORY_ERROR_DEFAULT_MSG from '../constants/errors';
+
+import LmsApiService from '../services/LmsApiService';
+import * as module from './grades';
+
+const {
+  formatMaxAssignmentGrade,
+  formatMinAssignmentGrade,
+  formatMaxCourseGrade,
+  formatMinCourseGrade,
+  formatGradeOverrideForDisplay,
+} = selectors.grades;
+
+export const defaultAssignmentFilter = 'All';
+
+export const fetchBulkUpgradeHistory = courseId => (
+  // todo add loading effect
```

Comment:
Is this todo actionable now or a future todo?

Reply:
I would argue out of scope.  Not sure the goal for it is described anywhere.

Reply:
Oh, like "add a loading spinner"? I wasn't sure if this was a "I am requesting we add new stuff" or a "I haven't done this yet but need to".

It it's an "I am requesting we add new stuff", then I'm okay to leave it out of scope.